contextproject-ms3/src/main/java/contextproject/models/Track.java

```diff
@@ -29,7 +31,10 @@
    * @param abPath
    *          Path of the mp3 file
    */
-  public Track(String abPath) {
+  public Track(String abPath, Hashtable<String, String> info) {
+
+    // String title, String artist, String album, long length, double bpm, String key, long
+    // firstBeat, int beatIntroStart, int beatsIntroLength, int beatOutroStart, int beatsOutroLength
```

Comment:
Would be nice if you could update the constructor comment with this information.